rules/config

```diff
@@ -30,7 +30,7 @@ SONIC_CONFIG_LOG_TO_FILES = y
 DEFAULT_USERNAME = admin
 
 # DEFAULT_PASSWORD - default password for installer build
-DEFAULT_PASSWORD = YourPaSsWoRd
+DEFAULT_PASSWORD = admin
```

Comment:
lguohan, 
Got it~ Could you help to discard the change of `rules/config`?  Not sure how to do so upon the pull request.